src/resources/Groups/GroupsInterfaces.ts

```diff
@@ -0,0 +1,71 @@
+export interface GroupModel {
+    id: string;
+    displayName: string;
+    invites?: InviteModel[];
+    members?: MemberModel[];
+    privileges?: PrivilegeModel[];
+    realms?: RealmModel[];
+    deletable?: boolean;
+    resourceId?: string;
+    groupsThatCanEdit?: IdAndDisplayNameModel[];
+    apiKeysThatCanEdit?: IdAndDisplayNameModel[];
+}
+
+export interface IdAndDisplayNameModel {
+    id: string;
+    displayName?: string;
+}
+
+export interface InviteModel {
```

Comment:
InviteModel is a tad complicated because it supports multiple ways to invite in the same model.

Invite by email only requires `email`. Invite by username requires only `username` to be set OR (`providerUsername` && `provider`)

Reply:
Alright I'll create InviteModel overloads

Reply:
Is there only 2 types of invites ?

Reply:
Yes 2 types.